docs/presigned-put-upload-via-browser.md

```diff
@@ -56,51 +56,50 @@ The client-side web application's user interface contains a selector field that
 
 <div id="status">No uploads</div>
 
-<script src="//code.jquery.com/jquery-3.1.0.min.js"></script>
 <script type="text/javascript">
   // `upload` iterates through all files selected and invokes a helper function called `retrieveNewURL`.
   function upload() {
-    // Reset status text on every upload.
-    $('#status').text(`No uploads`)
-    // Get selected files from the input element.
-    var files = $("#selector")[0].files
-    for (var i = 0; i < files.length; i++) {
-      var file = files[i]
-      // Retrieve a URL from our server.
-      retrieveNewURL(file, (file, url) => {
-        // Upload the file to the server.
-        uploadFile(file, url)
-      })
+        // Get selected files from the input element.
+        var files = document.querySelector("#selector").files
+        for (var i = 0; i < files.length; i++) {
+            var file = files[i];
+            // Retrieve a URL from our server.
+            retrieveNewURL(file, (file, url) => {
+                // Upload the file to the server.
+                uploadFile(file, url)
```

Comment:
There is just two very small semicolon issues left before approving, is it possible to add a semicolon at the end of this line?